README.md

```diff
@@ -182,14 +188,7 @@ For more information about configuration with ConfigMaps or Secrets, see [Config
 
 ## Viewing the status of published integrations
 
-After you publish a new integration, it appears in the **Apache Camel K Integrations** view in the Side Bar of the Explorer activity:
-
-![Apache Camel K Integrations view](images/camelk-integration-view.jpg)
-
-When you add or remove file-based integrations in the Explorer view, it automatically refreshes the list.
-
-**Note:** Refreshing the view sometimes is delayed as pods start. You might need to wait a few seconds. Optionally, to manually refresh the list, click the **Refresh** button.
-
+After you publish a new integration, it appears in the **AReuse
```

Comment:
Is it "A Reuse" or is this a typo?

Reply:
typo

Reply:
even worse, a part of the text was removed. seems my fingers had troubles :s